src/vs/workbench/contrib/chat/browser/chatStatus.ts

```diff
@@ -503,7 +504,7 @@ class ChatStatusDashboard extends Disposable {
 			if (quota.unlimited) {
 				usedPercentage = 0;
 			} else {
-				usedPercentage = Math.max(0, 100 - quota.percentRemaining);
+				usedPercentage = roundFloat(Math.max(0, 100 - quota.percentRemaining), 2);
```

Comment:
You may want to use [`Number.toPrecision`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision) instead, as you might get things like `40.999999999` to due floating point weirdness

Reply:
ah, sorry, that's a little different. I think the right thing is Intl.NumberFormat with maximumFractionDigits https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#minimumfractiondigits

Reply:
Should we just get rid of roundFloat as a helper then?

Reply:
I'm not sure its purpose in the RGBA logic, but for display I think Intl.NumberFormat is generally preferable. And it will also show the correct separators for the user's locale